setup.py

```diff
@@ -56,6 +56,8 @@
     "sentence-transformers",
     "open_clip_torch>=2.26.1",
     "peft",
+    "datasets[audio]>=1.4.0",
+    "tbb",
```

Comment:
Installing `datasets[audio]` leads to `numba` package also being installed. Because of this, `tbb` needs to be installed as well. This is because with `numbda` installed, but `tbb` not installed, the following error is raised inside INC: 
```
ValueError: No threading layer could be loaded.
HINT:
Intel TBB is required, try:
$ conda/pip install tbb
```

After both `numba` and `tbb` are installed, INC starts to pack bits relying on these packages. For example: https://github.com/intel/neural-compressor/blob/master/neural_compressor/torch/algorithms/weight_only/modules.py#L456-L457 

@IlyasMoutawwakil As we've agreed before, could you please invite somebody from INC side for a review?